Pipfile

```diff
@@ -56,6 +56,23 @@ dictdiffer = "*"
 setuptools = "*"
 markupsafe = "==2.0.1"
 imagesize = "*"
+cryptography = "*"
+typed-ast = "*"
+types-dateparser="*"
+types-pyyaml="*"
+types-requests="==2.26.3"# This is an open issue https://github.com/python/typeshed/issues/6893
+types-pytz="*"
+types-python-dateutil="*"
+types-pyvmomi="*"
+types-tabulate="*"
+types-paramiko="*"
+types-emoji="*"
+types-ipaddress="*"
+types-chardet="*"
+types-six="*"
+types-PyMySQL="*"
+types-protobuf="*"
```

Comment:
Do we explicitly need to mention all these? for instance`typed-ast` is installed with mypy

Reply:
mypy now doesnt ship with many of these stubs included. There's an option to run mypy so that it installs these stubs on the fly like I tried in 33fd2d288c3d69eea8e24065c2276322caff5d0e, but the pipeline here failed, and I didnt invest the time yet to understand why

Reply:
@jochman wdyt?

Reply:
@Itay4 thought the place for these dependencies were here. I thought you told me that the other day too @jochman ?